src/components/panel/panel.js

```diff
@@ -155,10 +156,11 @@ angular
  *     on when the panel closes. This is commonly the element which triggered
  *     the opening of the panel. If you do not use `origin`, you need to control
  *     the focus manually.
- *   - `groupName` - `{string=}`: Name of a panel group. This group name is used
- *     for configuring the number of open panels and identifying specific
- *     behaviors for groups. For instance, all tooltips could be identified
- *     using the same groupName.
+ *   - `groupName` - `{(string|string[])=}`: A single name of a panel group. Can
```

Comment:
nit: You're saying "a single name of a group" and "can also be an array" in two consecutive sentences. Might be better to say something along the lines of "A group name or an array of groups".